requirements/reproducible-linux.txt

```diff
@@ -126,6 +131,10 @@ cffi==1.15.1 \
     --hash=sha256:fa6693661a4c91757f4412306191b6dc88c1703f780c8234035eac011922bc01 \
     --hash=sha256:fcd131dd944808b5bdb38e6f5b53013c5aa4f334c5cad0c72742f6eba4b73db0
     # via cryptography
+cfn-lint==0.72.2 \
```

Comment:
Are there any dependencies which is dynamically imported? If so we might need to add them to hidden imports information for pyinstaller.

Reply:
https://github.com/aws/aws-sam-cli/pull/4476